ci-docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG TARGET_DOCKER_PLATFORM
+
+FROM library/debian:stretch
+
+RUN apt-get update && apt-get install -y qemu-user-static
+
+FROM $TARGET_DOCKER_PLATFORM/debian:stretch
+
+ARG HOST_DEBIAN_PLATFORM
+
+COPY --from=0 /usr/bin/qemu-* /usr/bin/
+
+RUN dpkg --add-architecture $HOST_DEBIAN_PLATFORM
+
+RUN apt-get update && apt-get install -y openjdk-8-jdk-headless:$HOST_DEBIAN_PLATFORM
+
+ENV PATH "/usr/lib/jvm/java-8-openjdk-$HOST_DEBIAN_PLATFORM/bin:$PATH"
+
+ENV SBT_LAUNCHER_VERSION 1.2.0
```

Comment:
Would make sense to keep this in sync with the version in `project/build.properties`, i.e. 0.13.17?